packages/client/src/components/chart.vue

```diff
@@ -7,8 +7,14 @@
 </div>
 </template>
 
-<script lang="ts">
-import { defineComponent, onMounted, ref, watch, PropType, onUnmounted, shallowRef } from 'vue';
+<script lang="ts" setup>
+/* eslint-disable id-denylist --
+  Chart.js has a `data` attribute in most chart definitions, which triggers the
+  id-denylist violation when setting it. This is causing about 60+ lint issues.
```

Comment:
I understand what you are doing here, but would it not be better to limit the area covered by this a bit more? The issue with `data` does not occur until line 80 or so, and that is only in the type definition. Not sure if eslint picks that up.

Reply:
Well, the initial type information thing is not picked up by `id-denylist`, but everytime an assignment to an object property `data` is happening this is triggered, causing the many errors.

The first actual lint coming up is on line 215 probably. So I get your point, but a lot of the early code in the script tag is just component setup and imports in general, which is why I opted to put the comment right at the top so I don't have a huge comment + explainer somewhere in the middle of it.

I actually wanted to only remove `data` from `id-denylist` for this file, but the ESLint comment to adjust the rule didn't really work and only fully disabling with with the `eslint-disable` comment had an effect. :/